cli/index.js

```diff
@@ -0,0 +1,105 @@
+// NPM Modules
+import meow     from 'meow';
+import stdin    from 'get-stdin';
+import textr    from 'textr';
+
+// Native Modules
+import path     from 'path';
+import fs       from 'fs';
+
+/**
+ * CLI app instance of Meow
+ * More: https://www.npmjs.com/package/meow
+ * @param  {Object|Array|String} Text that will be write on --help or -h
+ * @param  {Object}              Minimist options (github.com/substack/minimist)
+ * @return {Object}              A Meow CLI helper object
+ */
+const cli = meow(`
+  Usage
+    $ textr
+
+  Options
+    -t, --transforms    Array of transformers
+    -o, --out-file      Write output to file
+    -l, --locale        ISO 639 locale codes (en-us as default)
+    -h, --help          Show this message
+
+  Examples
+    $ textr foo.md -t typographic-quotes -t typographic-quotes
+    $ textr foo.md -t typographic-single-spaces,typographic-quotes
+    $ cat foo.md | textr -l ru --transforms=typographic-single-spaces
+    $ cat foo.md | textr -o bar.md
+`,
+  {
+    default: {
+      transforms: [],
+    },
+    alias: {
+      t: 'transforms',
+      o: 'out-file',
+      l: 'locale',
+      h: 'help'
+    }
+  }
+);
+
+/**
+ * Resolve file name with current dir
+ * @param  {String} file Any relative filename
+ * @return {String}      Resolved path to file
+ */
+const cwd = file =>
+  path.resolve(process.cwd(), file);
+
+/**
+ * Render text string throught textr transformer
+ * @param  {String} text Text that will be processed
+ * @param  {Array}  tfs  Array of transformers
+ * @return {Void}        Write into strout or file and exit
+ */
+const render = (text, tfs = [], locale = 'en-us') => {
+  const res = (textr({ locale }).use.apply(null, tfs))(text);
+  if (cli.flags.outFile) {
+    try {
+      fs.writeFileSync(cwd(cli.flags.outFile), res, 'utf8');
```

Comment:
In cli apps better to use blocking io or use promises. Shell scripts exec from top to bottom, so async seem to be inconvenient.
https://strongloop.com/strongblog/node-js-v0-12-shell-programming-synchronous-child-process/